development/webpack/webpack.config.ts

```diff
@@ -54,6 +59,7 @@ if (args.manifest_version === 3) {
 // #endregion short circuit for unsupported build configurations
 
 const context = join(__dirname, '../../app');
+const projectRoot = join(__dirname, '../../'); // While ../../app is the main dir for the webpack build to use as context, the project root where package.json is one level up. This discrepancy needs to be explained to LavaMoat plugin as it's searching for the package.json in the compilator.context by default.
```

Comment:
Shouldn't the plugin search upwards in the directories until it finds a `package.json`?